web-jehlomat/src/screens/Nalezy/Components/DarkButton.ts

```diff
@@ -0,0 +1,26 @@
+﻿import { styled } from '@mui/system';
+
+const DarkButton = styled('button')({
```

Comment:
Same as `Button` component. In general; introduce general button component with `component` prop, which one can define underlying element (anchor, button). After that the `DarkButton` should be only the `styled` component with different colors.